src/components/PersonalData/TextField/TextField.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "./TextField.css";
+const TextField = props => {
+  return (
+    <form>
+      <div className="form-group bc-color">
+        <label htmlFor="exampleInputName">Email</label>
+        <input
+          className="form-control"
+          id="email"
+          value={props.email}
+          disabled={true}
+        />
+        <label htmlFor="exampleInputName">Imię</label>
```

Comment:
could you put all texts in variables? it will be easier if we decide to add different language for app